### PR TITLE
Preparing Release for Schema Registry

### PR DESCRIPTION
**Pull request by conniey**
# Description

Preparing release for Schema Registry
* azure-data-schemaregistry
* azure-data-schemaregistry-apacheavro

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/schemaregistry/azure-data-schemaregistry/CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 - Added `retryOptions` to `SchemaRegistryClientBuilder`.
 
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+#### Dependency Updates
 
-### Other Changes
+- Update `azure-core` dependency to `1.26.0`.
```
Comment (srnagar) [inline sdk/schemaregistry/azure-data-schemaregistry/CHANGELOG.md:14]: We should also include the netty version update here.

**Review by srnagar — APPROVED** [review]

**Comment by hernandezmshawntae** [comment]
Please and Thank You.

**Review by hernandezmshawntae — COMMENTED** [review]
Please and Thank You.

**Comment by hernandezmshawntae** [comment]
please and thank you

**Comment by hernandezmshawntae** [comment]
privacy issues data loss

File: sdk/schemaregistry/azure-data-schemaregistry/CHANGELOG.md (hunk 1 above)
Comment (hernandezmshawntae) [reply sdk/schemaregistry/azure-data-schemaregistry/CHANGELOG.md:14]: privacy issues data loss #fraud

**Pull request merged**